pkg/plugin/response.go

```diff
@@ -0,0 +1,105 @@
+package plugin
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+	"github.com/grafana/grafana-plugin-sdk-go/data"
+)
+
+type Result struct {
+	Metric Metric  `json:"metric"`
+	Values []Value `json:"values"`
+	Value  Value   `json:"value"`
+}
+
+type Metric map[string]string
+
+type Value [2]interface{}
+
+type Data struct {
+	ResultType string   `json:"resultType"`
```

Comment:
It doesn't look like ResultType is checked somewhere. Why would you need that? 
It also looks like `scalar` type of response will fail - see https://github.com/VictoriaMetrics/VictoriaMetrics/issues/2607

Reply:
reimplement that logic, add different response types and collection of the data frames